codejail.profile

```diff
@@ -60,7 +60,7 @@ profile codejail_service flags=(mediate_deleted) {
 
     # Allow sending a kill signal to the codejail_sandbox subprofile when the execution
     # runs beyond time limits.
-    signal (send) set=(kill) peer=codejail_service//codejail_sandbox,
+    signal (send) set=(kill) peer=openedx_codejail_service//codejail_sandbox,
```

Comment:
Are there bugs that we missed where `codejail_sandbox` references need to be updated?

Reply:
Not following -- I don't think there are any functional changes in the renaming part of this diff.

Reply:
Oops. I got confused. That said, can you better explain why you wanted the `openedx_` prefix here, and why you don't want it for `openedx_codejail_sandbox`?

Reply:
Oh, the idea there is just that the namespacing is only important on the outermost profile. The inner one only shows up in the context of the outer one; it doesn't really have an independent existence. To illustrate, here's what I see when I ask AppArmor about its current profiles (and processes) and filter it to just codejail-related things:

```
$ sudo aa-status | grep codejail
   openedx_codejail_service
   openedx_codejail_service//codejail_sandbox
```